src/pages/_app.tsx

```diff
@@ -1,9 +1,22 @@
+import type { ReactElement, ReactNode } from 'react';
+import type { NextPage } from 'next';
 import { AppProps } from 'next/app';
 import '../styles/globals.css';
 import '../../styles.css';
 
-function MyApp({ Component, pageProps }: AppProps) {
-  return <Component {...pageProps} />;
+type NextPageWithLayout = NextPage & {
+  // eslint-disable-next-line no-unused-vars
+  getLayout?: (page: ReactElement) => ReactNode;
+};
+
+type AppPropsWithLayout = AppProps & {
+  Component: NextPageWithLayout;
+};
+
+function MyApp({ Component, pageProps }: AppPropsWithLayout) {
+  const getLayout = Component.getLayout ?? (page => page);
```

Comment:
`src/pages/index.tsx` だけが今回作成した Layout コンポーネントを使わないので、以下のようにすると良いかなと思いましたがいかがでしょう？
```suggestion
  const getLayout = Component.getLayout ?? (page => <Layout>page</Layout>);
```

Reply:
こちら変更してみたところ、各ページの表示がこのようになってしまいました。
<img width="1680" alt="スクリーンショット 2021-11-03 16 50 32" src="https://user-images.githubusercontent.com/60058000/140025474-09509fd4-7cec-48a7-9c0a-e3c1026c5ec6.png">

_app.tsxは、17行目をご提案いただいたコードに変更し、Layoutコンポーネントのimportを追加しましたが、他に変更するところはありますでしょうか？
<img width="1680" alt="スクリーンショット 2021-11-03 16 47 09" src="https://user-images.githubusercontent.com/60058000/140025979-4c1362a9-d9cb-4259-96ce-f7ccccdf0109.png">

Reply:
あ、すみません、 `page` が文字列になってしまっているので以下ですね 🙏 
```js
const getLayout = Component.getLayout ?? (page => <Layout>{page}</Layout>);
```

あとは `src/pages/broadcasts/index.tsx` の 15行あたりの `getLayout` の指定は不要になるのではないでしょうか？

Reply:
あとは以下のように変更すればいい感じになりました！

```diff
diff --git a/src/pages/_app.tsx b/src/pages/_app.tsx
index badd72e..c571164 100644
--- a/src/pages/_app.tsx
+++ b/src/pages/_app.tsx
@@ -1,6 +1,7 @@
 import type { ReactElement, ReactNode } from 'react';
 import type { NextPage } from 'next';
 import { AppProps } from 'next/app';
+import { Layout } from '../components/Layout';
 import '../styles/globals.css';
 import '../../styles.css';
 
@@ -13,7 +14,7 @@ type AppPropsWithLayout = AppProps & {
 };
 
 function MyApp({ Component, pageProps }: AppPropsWithLayout) {
-  const getLayout = Component.getLayout ?? (page => page);
+  const getLayout = Component.getLayout ?? (page => <Layout>{page}</Layout>);
 
   return getLayout(<Component {...pageProps} />);
 }
diff --git a/src/pages/index.tsx b/src/pages/index.tsx
index 3b521b3..d413e6e 100644
--- a/src/pages/index.tsx
+++ b/src/pages/index.tsx
@@ -1,6 +1,7 @@
 import Head from 'next/head';
 import Image from 'next/image';
 import Link from 'next/link';
+import type { ReactElement } from 'react';
 
 export default function Home() {
   return (
@@ -46,3 +47,5 @@ export default function Home() {
     </div>
   );
 }
+
+Home.getLayout = (page: ReactElement) => page;

```